### ISSUE TITLE
UnhandledPromiseRejectionWarning: Error: Request failed with status code 400

### ISSUE THREAD

**Issue by Bikeshtri** [slackapi/slack-github-action#7]
### Description
Getting error while using 
```
- name: Send GitHub Action trigger data to Slack workflow
  id: slack
  uses: slackapi/slack-github-action@v1.14.0
  env:
    SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [x] bug
- [ ] enhancement (feature request)
- [ ] question
- [ ] documentation related
- [ ] example code related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [ ] I've read and understood the [Contributing guidelines](https://github.com/slackapi/slack-github-action/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [ ] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [ ] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

```
node:1549) UnhandledPromiseRejectionWarning: Error: Request failed with status code 400
no custom payload was passed in, using default payload that triggered the GitHub Action
    at createError (/home/runner/work/_actions/slackapi/slack-github-action/v1.14.0/dist/index.js:8314:15)
    at settle (/home/runner/work/_actions/slackapi/slack-github-action/v1.14.0/dist/index.js:8573:12)
axios post failed, double check the payload being sent includes the keys Slack expects
    at IncomingMessage.handleStreamEnd (/home/runner/work/_actions/slackapi/slack-github-action/v1.14.0/dist/index.js:7703:11)
{
    at IncomingMessage.emit (events.js:215:7)
  action: 'completed',
    at endReadableNT (_stream_readable.js:1184:12)
  organization: {
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
    avatar_url: 'https://avatars.githubusercontent.com/u/********?v=4',
(node:1549) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
    description: 'Publisher of academic abesjgr, books, media and information',
(node:1549) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    events_url: 'https://api.github.com/orgs/
```

#### Reproducible in:

package version:

node version:

OS version(s):

#### Steps to reproduce:

1.
2.
3.

#### Expected result:

What you expected to happen

#### Actual result:

What actually happened

#### Attachments:

Logs, screenshots, screencast, sample project, funny gif, etc.

**Comment by chinochao** [comment]
I am having the same issue. Any help will be appreciated.

`Run slackapi/slack-github-action@v1.15.0
5
no custom payload was passed in, using default payload that triggered the GitHub Action
6
axios post failed, double check the payload being sent includes the keys Slack expects
.....
}
575
    at createError (/runner_dir/_actions/slackapi/slack-github-action/v1.15.0/dist/index.js:8314:15)
576
    at settle (/runner_dir/_actions/slackapi/slack-github-action/v1.15.0/dist/index.js:8573:12)
577
    at IncomingMessage.handleStreamEnd (/runner_dir/_actions/slackapi/slack-github-action/v1.15.0/dist/index.js:7703:11)
578
    at IncomingMessage.emit (events.js:215:7)
579
    at endReadableNT (_stream_readable.js:1184:12)
580
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
581
(node:23804) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
582
(node:23804) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`

**Comment by seratch** [comment]
Hi @Bikeshtri, I'm sorry for our late response here. Your issue description does not have the message body part but I'm guessing that you have `blocks` in it, right? If so, the issue is due to the same reason with https://github.com/slackapi/slack-github-action/issues/11. The issue will be resolved in future versions.

**Comment by stevengill** [comment]
This landed in `v1.16.0`. Going to close the issue, but feel free to re-open if it isn't solved.
